kwc-share-card.html

```diff
@@ -109,27 +114,15 @@
                     type: Object,
                     value: null
                 },
-                /** TODO: Encode a default avatar svg and use it instead */
                 /**
                  * Image to fallback in case user doesn't have an avatar.
                  * @type {String}
                  */
                 defaultAvatar: {
                     type: String,
-                    value: 'https://hoc-staging.kano.me/assets/images/default-avatar.svg'
-                },
-                /**
-                 * TODO: Encode a default staff picked svg badge, use it instead
-                 * and allow user to pass a custom one
-                 */
-                /**
-                 * Flags if this share has been staff picked.
-                 * @type {Boolean}
-                 */
-                staffPicked: {
-                    type: Boolean,
-                    value: false,
-                    reflectToAttribute: true
+                    value: () => {
+                        return window.KwcShareCardAssets.assets['defaultAvatar'];
```

Comment:
Do we want to scope this under `Kano` as with other global properties? We could potentially re-use and share these – `Kano.Components.Assets.DefaultAvatar` or something? Maybe that's a recipe for headaches later on though...

Reply:
I honestly have no strong opinion on this apart from follow a pattern. I put this straight on the window because that is how I saw it done [here](https://github.com/KanoComponents/kwc-challenge-set/blob/master/scripts/progressIcons.html) and I have already done [this](https://github.com/KanoComponents/kwc-challenge-set/blob/master/scripts/getMedalButton.html)... While namespacing it under `Kano` sounds like a good idea, I am not so keen on doing the same thing in many ways :s